public/js/stores/GraphStore.js

```diff
@@ -0,0 +1,31 @@
+import { EventEmitter } from 'events'
+import dispatcher from '../dispatcher'
+
+class GraphStore extends EventEmitter {
+  constructor() {
+    super()
+    this.points = [['TempData', 'TempData'], [1, 1]]
+  }
+
+  setPoints(dataPoints) {
+    this.points = dataPoints
+    this.emit('graphUpdate')
+  }
+
+
+  getDataPoints() {
+    return this.points
+  }
+
+  handleAction(action) {
+    switch (action.type) {
+      case 'UPDATE_GRAPH':
```

Comment:
Should add a block here to just in case you declare variables in the case's scope 
```javascript 
case 'UPDATE_GRAPH': {
  // code
  break
}
```